tests/test_asb_encode_binding.yml

```diff
@@ -0,0 +1,79 @@
+---
+- hosts: localhost
+  vars:
+    binding_file_path: /var/tmp/bind-creds
+  roles:
+    - ansibleplaybookbundle.asb-modules
+  tasks:
+    - name: prepare test for clean run
+      file:
+        path: "{{ binding_file_path }}"
+        state: absent
+
+    - name: encode bind credentials
+      asb_encode_binding:
+        fields:
+          TEST_VAR_1: "test value 1"
+          TestValue2: 2
+      register: apb_result
+
+    - debug:
+        var: apb_result
+
+    - name: decode encoded value returned
+      shell: 'echo "{{ apb_result.encoded_fields }}" | base64 -d'
```

Comment:
Until we come up with a strong use case, I would lean towards no.  With user facing APIs is that it's easy to give and hard to take away.  This is just for testing right now.